.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 .env
+.env.sample
+.commitmsg
```

Comment:
Any reason for this?

Reply:
Oh, I overlooked that apparently. 

.commitmsg was purely so I could edit my commits while I made each change, and .env.sample was just me paranoidly making sure I didn't push any changes I might've made there.

Neither of them are critically important.

Reply:
Awesome, thank you, go ahead and remove this bit since it's not relevant.

Reply:
Done